tests/e2e/fixtures/storage_fsx_dependencies.py

```diff
@@ -117,7 +117,8 @@ def on_delete():
         )
 
         # delete the security group
-        ec2_client.delete_security_group(GroupId=sg_id)
+        # TODO: This needs to be fixed. we need to wait for fsx volume to be deleted but fsx does not provided a deleted status, needs a workaround. 
+        # ec2_client.delete_security_group(GroupId=sg_id)
```

Comment:
can we add a sleep command for now
if we dont delete the group not sure how long till we hit the service limit on num of sgs

Reply:
I thought of that but I do know a cleaner way to fix this - documented it here - https://github.com/awslabs/kubeflow-manifests/issues/200

Any amount of sleep is not reliable and I think this step does take a while. Since we are not running this test in any pipeline at the moment we should be ok and I will fix this asap post release.